dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/validator/event/SecurityOwnershipValidatorTest.java

```diff
@@ -489,6 +489,8 @@ private TrackedEntityInstance getTEIWithNoProgramInstances() {
     trackedEntityInstance.setProgramInstances(Sets.newHashSet());
     trackedEntityInstance.setTrackedEntityType(trackedEntityType);
 
+    assertIsEmpty(reporter.getErrors());
```

Comment:
why is this assertion in here and not in a test?

Reply:
Good catch, I guess a merging problem